datacommons/test/datacommons_test.py

```diff
@@ -74,8 +74,8 @@ def test_query(self):
         'header': ['name', 'area_sq_mi']
     })
     expected_df = pd.DataFrame({
-        'name': [['Oceania']],
-        'area_sq_mi': [['3292000']]
+        'name': ['Oceania'],
+        'area_sq_mi': ['3292000']
     })
```

Comment:
Could you pls add repeated property to validate those become multiple rows.

Reply:
Fixed f74cdd6 !